scss/_variables.scss

```diff
@@ -44,7 +44,7 @@ $popup-padding-x: 0 !default;
 $popup-padding-y: 0 !default;
 
 $header-padding-x: $padding-x !default;
-$header-padding-y: $padding-y !default;
+$header-padding-y: $padding-x !default;
```

Comment:
I don't think it's predictable at all for customers to change vertical padding with `$padding-x`.

If the padding is always equal on all sides, then we don't need two variables. If it can be different (maybe in theme-bootstrap it is?), then we should maintain the relationships between `x` and `y` variables, in order for customization to be predictable.